docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,740 @@
+        # 4GB download!
+        sudo python ~/src/on-tools/scripts/setup_iso.py \
+        /tmp/CentOS-7-x86_64*.iso /var/mirrors \
+        --link=/home/vagrant/src
+
+- add the workflow to install CentOS for this specific SKU::
+
+    cd ~/src/rackhd/example
+    # make sure you're in the example directory to reference the sample JSON correctly
+
+    curl -H "Content-Type: application/json" \
+    -X PUT --data @samples/vQuanta_install_centos.json \
+    http://localhost:9090/api/1.1/workflows
+
+The `vQuanta_install_centos.json` sample
+
+.. code-block:: JSON
+
+    {
+        "friendlyName": "vQuanta D51 Default Install CentOS",
+        "injectableName": "Graph.DefaultVQuanta.InstallCentOS",
+        "options": {
+          "defaults": {
+            "version": "7",
+            "dnsServers": ["8.8.8.8", "8.8.4.4"],
+            "repo": "{{api.server}}/Centos/7.0",
+            "rootPassword": "rootpassword",
+            "users": [{
+              "name": "rackhd",
+              "password": "rackhd123",
+              "uid": 1010
+            }]
+            },
+          "install-centos": {
+            "schedulerOverrides": {
+              "timeout": 3600000
+            }
+          }
+        },
+        "tasks": [
+            {
+                "label": "create-ipmi-obm-settings",
+                "taskDefinition": {
+                    "friendlyName": "Create IPMI OBM settings",
+                    "injectableName": "Task.Obm.Ipmi.CreateSettings",
+                    "implementsTask": "Task.Base.Obm.Ipmi.CreateSettings",
+                    "options": {
+                        "ipmichannel": null,
+                        "user": "admin",
+                        "password": "admin"
+                    },
+                    "properties": {
+                    }
+                }
+            },
+            {
+                "label": "install-coreos",
+                "taskName": "Task.Os.Install.CentOS",
+                "waitOn": {
+                    "create-ipmi-obm-settings": "succeeded"
+                }
+            }
+        ]
+    }
+
+- add the SKU definition for our simulated hardware::
+
+    cd ~/src/rackhd/example
+    # make sure you're in the example directory to reference the sample JSON correctly
+
+    curl -H "Content-Type: application/json" \
+    -X POST --data @samples/vQuanta_d51_sku.json \
+    http://localhost:9090/api/1.1/skus
+
+The `vQuanta_d51_sku.json` sample
+
+.. code-block:: JSON
+
+    {
+        "name": "vQuanta D51 SKU",
+        "discoveryGraphName": "Graph.DefaultVQuanta.InstallCentOS",
+        "discoveryGraphOptions": {},
+        "rules": [
+            {
+                "path": "dmi.System Information.Product Name",
+                "equals": "D51B-2U (dual 10G LoM)"
+            },
+            {
+                "path": "dmi.System Information.Serial Number",
+                "equals": "SerialNumber"
```

Comment:
Yeah `SerialNumber` - it's a quirk of the InfraSim instance that I decided to take advantage of in case someone had a Quanta sku loaded for some reason as well - that way this demonstration SKU would not conflict with any of the actual Quanta SKU definitions.